Documentation/coding-guidelines/adding-api-guidelines.md

```diff
@@ -0,0 +1,124 @@
+#Add APIs
+
+- [Determining versions and targets](#determining-versions-and-targets)
+- [Making the changes in repo](#making-the-changes-in-repo)
+- [FAQ](#faq)
+
+##Determining versions and targets
+
+1. [Determine what library](#determine-what-library) the API goes into.
```

Comment:
All reference assemblies have a matching implementation assembly even if it is just a pure facade. I don't think there is really any assumption in this doc about them needing to be the same, but perhaps I'm missing something.